src/js/static.js

```diff
@@ -4,6 +4,22 @@ const PatientStatusIcons = {
   done: 'check-circle',
 };
 
+// NOTE: id: 'admin' purposefully not included
+// NOTE: Intentionally not i18n'd: may move to BE
+const ACCESS_TYPES = [
+  {
+    id: 'employee',
+    name: 'Employee',
+    details: 'can access Workspace but not Admin',
+  },
+  {
+    id: 'manager',
+    name: 'Manager',
+    details: 'can update Workplace and Admin',
```

Comment:
Are "Workplace" and "Workspace" two different things?

Reply:
🤦